include/grpc/grpc.h

```diff
@@ -506,6 +506,9 @@ GRPCAPI char* grpc_channelz_get_top_channels(intptr_t start_channel_id);
 /* Gets all servers that exist in the process. */
 GRPCAPI char* grpc_channelz_get_servers(intptr_t start_server_id);
 
+/* Returns a single Server, or else a NOT_FOUND code. */
+GRPCAPI char* grpc_channelz_get_server(intptr_t server_id);
```

Comment:
I assume we're also going to need to add this method to the C++ service implementation?

Reply:
Yep, I was going to do that in a follow up with the appropriate testing in the channelz_service_test file